pages/user-interfaces.html

```diff
@@ -55,82 +43,98 @@
 </p>
 
 <p>
-An experimental alternative to the editors mentioned above is the
-computational notebook interface provided by Jupyter.
-A <a href="https://github.com/EugeneLoy/coq_jupyter/">Jupyter kernel
-for Coq</a> is available if you want to try this interface, for
-instance for pedagogical or readability purposes.  One of the
-advantages of using a Jupyter notebook is that you can save and
-display a selection of intermediate proof steps, which your reader
-will see without the need to reexecute the document.
+Otherwise, <a href="https://coq.inria.fr/refman/practical-tools/coqide.html">CoqIDE</a> is
+a <strong>stand-alone</strong> interface
+developed and distributed alongside Coq.
+CoqIDE comes with standard basic Emacs-like edition bindings
+and with some specific Coq-related features (e.g. for searching, printing, debugging) but, as an editor, it is
+not as complete as a general-purpose editor could be (for instance it
+does not have automatic indentation).
+</p>
+
+<p>
+As a way to try Coq without installing anything, you can also
+use <a href="https://jscoq.github.io/">JsCoq</a> which loads Coq
+entirely in <strong>your browser</strong>.  However, it is limited to conduct
+actual projects: e.g., it lacks access to the VM and native computing
+machineries of Coq, and running in the browser is slower.
 </p>
 
 </div>
 
 <div class="frameworklinks">
 <ul>
-<li><a class="extlink" href="https://github.com/coq-community/vscoq">VsCoq (VS Code)</a></li>
-<li><a class="extlink" href="https://github.com/ejgallego/coq-lsp">coq-lsp (VS Code)</a></li>
+<li><a class="extlink" href="https://github.com/coq-community/vscoq/tree/vscoq1">VsCoq 1 (VS Code)</a></li>
 <li><a class="extlink" href="https://proofgeneral.github.io/">Proof General (Emacs)</a></li>
 <li><a class="extlink" href="https://github.com/whonore/Coqtail">Coqtail (Vim)</a></li>
+<li><a href="refman/practical-tools/coqide.html">CoqIDE</a></li>
+<li><a class="extlink" href="https://jscoq.github.io/">JsCoq</a></li>
 </ul>
 </div>
 </div>
 
 <div class="framework">
-<div class="frameworklabel">Standalone interfaces</div>
+<div class="frameworklabel">Towards a modular design of Coq user interfaces based on the LSP protocol</div>
```

Comment:
Same, it fits for me. I would prefer to keep "design", e.g. by using "LSP-based". And otherwise, ok for your shortening.

Reply:
Then we may keep "toward" instead of "experimental" as well, I suppose. The following fits for me:
```suggestion
<div class="frameworklabel">Towards an LSP-based modular design of Coq user interfaces</div>
```